### ISSUE TITLE
No space between closed poll and thank you messages

### ISSUE THREAD

**Issue by danis039** [Automattic/crowdsignal-forms#185]
When a user votes on a poll which has the option "One response per computer" enabled and later when this poll has been closed, the message displayed by default is the following: "This Poll is ClosedThanks For Voting!". As seen in the screenshot below, the messages are shown together with no spacing between them.  

![image](https://user-images.githubusercontent.com/11977521/137531957-0f11dc85-8168-45d4-b0e7-c12e25330b8c.png)

As these messages are not wrapped in `span` elements it's not really possible to style them with CSS.

Any possibility to have a fix for this, please?

Thanks in advance! Loving the plugin :)

**Comment by bobmatyas** [comment]
I don't believe this behavior still exists. It looks like it was addressed on https://github.com/Automattic/crowdsignal-forms/pull/269

Screenshot:

<img width="3674" height="2090" alt="Image" src="https://github.com/user-attachments/assets/66e0f506-d2f6-4bf5-8796-55704327801d" />
